data/tabular/qm9/transform.py

```diff
@@ -0,0 +1,54 @@
+import datasets
+import pandas as pd
+
+DATASET_NAME = "qm9"
+
+
+def prepare_data():
+    dataset_name = "n0w0f/qm9-csv"
+    split_name = "train"  # data without any split @ hf
+    filename_to_save = "data_clean.csv"
+
+    # Load the dataset from Hugging Face
+    dataset = datasets.load_dataset(dataset_name, split=split_name)
```

Comment:
That is super awesome! Thanks for directly uploading it to HF! 💯 Very happy to see that!